shared/cfg/guest-os/Linux/RHEL/8/8.devel.cfg

```diff
@@ -0,0 +1,9 @@
+- devel:
+    # Uses defaults set in 8.cfg
+    # Adding x.y versions should only require changing the kickstart
+    # and adding per-arch image shas.
+    image_name = images/${os_variant}devel-${vm_arch_name}
+    unattended_install.cdrom:
+        cdrom_cd1 = isos/linux/RHEL-8-devel-${vm_arch_name}.iso
+    unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+        unattended_file = unattended/RHEL-8-devel.ks
```

Comment:
Would you add `RHEL-8-devel.ks` too ?  or a separate commit/patch ?

Reply:
hi @yanan-fu,
 we - SPICE-QE use custom kickstart file for RHEL-8. 
I would create ks file for RHEL-8 here as well, but `Appstream` repo is needed to install `@Workstation` group. I do not know how to include it without explicitly specifying our internal path to the repo. Any hint?

Reply:
Oh, good point, I forgot about the kickstart file. Would you please add it with commented `repo` entry with a comment that one should add his own url? Besides, specifying url directly from kickstart does not work: https://bugzilla.redhat.com/show_bug.cgi?id=1649359

Reply:
@rdkdd  AFAIK,  currently, RHEL8 iso include both `BaseOs` and `AppStream`, no need to add AppStream repo extra if install from iso.
Also,  you can refer this commit 81f9de5b5261ff3ff5432109ba1bc80a93d3c500. We support to add extra repo in ks:
1. Define `kickstart_extra_repos` in test.
2. Add `KVM_TEST_REPOS` in ks.